modules/data_warehouse/versions.tf

```
@@ -37,7 +37,7 @@ terraform {
       version = ">= 0.9.1"
     }
   }
-  required_version = ">= 0.13"
+  required_version = ">= 1.3"
```

Keeping this at 0.13 since it has some other dependencies.
```suggestion
  required_version = ">= 0.13"
```